oneview_redfish_toolkit/app.py

```diff
@@ -233,6 +235,22 @@ def not_implemented(error):
             status=status.HTTP_501_NOT_IMPLEMENTED,
             mimetype='application/json')
 
+    if util.config['redfish']['authentication_mode'] == 'conf':
+        # Loading scmb connection
+        if scmb.check_cert_exist():
+            logging.info('SCMB certs already exists testing connection...')
+        else:
+            logging.info('SCMB certs not found. Generating/getting certs....')
+            scmb.get_cert()
+            logging.info('Got certs. Testing connection...')
+        scmb.test_cert()
```

Comment:
Should this be called here?  It seems like if you have certs then you should connect and start listening for messages.  No need to 'test' the certs.

Reply:
If you are confident that cert tests are unnecessary I can remove the test. I took in consideration that the certs can be modified by user, os the download may be corrupted. If you think it's a extreme care I'll remove the call and the function.